fv/host_port_test.go

```diff
@@ -41,6 +42,7 @@ func RunFelix(etcdIP string) *containers.Container {
 	return containers.Run("felix-fv",
 		"--privileged",
 		"-e", "CALICO_DATASTORE_TYPE=etcdv2",
+		"-e", "FELIX_LOGSEVERITYSCREEN=debug",
```

Comment:
I guess this is OK for now, with all existing tests being low-scale and functional.  If and when we migrate the k8sfv scale tests here, we will probably need to parameterize this.